pkg/minikube/assets/addons.go

```diff
@@ -47,6 +47,11 @@ func NewAddon(assets []*BinAsset, enabled bool, addonName string) *Addon {
 	return a
 }
 
+// GetAddonName get the addon name
+func (a *Addon) GetAddonName() string {
```

Comment:
Thanks for all the changes, please change this to Name(). I think after this change it will be good to go !

Reply:
Done. Thanks